### PR TITLE
Use EF library from currently active project to generate views and ddl;

### PR DESCRIPTION
**Pull request by AlexKichkailo**
Fix for the #issue 22

### CODE REVIEW

File: src/PowerTools/Properties/Resources.cs [hunk 1]
```diff
@@ -221,6 +221,14 @@ internal static string UnsupportedVersion
             get { return EntityRes.GetString(EntityRes.UnsupportedVersion); }
         }
 
+        /// <summary>
+        /// A string like "Failed to load EntityFramework assembly from currently active project. Using EntityFramerork from EF Tools installed in Visual Studio."
```
Comment (ErikEJ) [inline src/PowerTools/Properties/Resources.cs]: Typo

File: src/PowerTools/Properties/Resources.resx [hunk 2]
```diff
@@ -199,6 +199,9 @@
     <value>You are using a version of the Entity Framework that is not supported by the Power Tools. Please upgrade to Entity Framework 4.2 or later.</value>
     <comment>## ExceptionType=InvalidOperationException</comment>
   </data>
+  <data name="UseReferencedEntityFrameworkAssemblyError" xml:space="preserve">
+    <value>Failed to load EntityFramework assembly from currently active project. Using EntityFramerork from EF Tools installed in Visual Studio.</value>
```
Comment (ErikEJ) [inline src/PowerTools/Properties/Resources.resx]: Typo

**Comment by ErikEJ** [comment]
Looks very promising! Thanks (just noted a couple of typos)

**Comment by AlexKichkailo** [comment]
I just did more testing and discovered that the fix does not work. I am getting Type mismatch exception when calling InvokeMember("ObjectContext"...) method. Will try to fix it a bit later.

**Comment by AlexKichkailo** [comment]
I think I will give up on this. The EntityFramework.dll from EF Tools is loaded into the Visual Studio's default app domain so in order to use the EntityFramework.dll from the currently active project, I need to load all project assemblies into a separate appDomain; or try to specify assembly binding information. This really complicates the code and may introduce other issues. So I will close the PR and wait for an update of EF tools.

**Pull request closed without merging**